### ISSUE TITLE
[redmine] Images and Thumbnail are not interpreted in table while exporting PDF

### ISSUE THREAD

**Issue by marutosi** [naitoh/rbpdf#12]
http://www.redmine.org/issues/19364

**Comment by naitoh** [comment]
This problem is img tag's width/height style attribute bug.

```
Sample Code Result
<table>
    <tr>
      <th>Example</th>
      <th>data</th>
    </tr>
    <tr>
      <td>image with thumbnail</td>
      <td><a class="thumbnail" title="redmine.jpeg" href="http://192.168.56.22:3001/300/attachments/4"><img alt="redmine.jpeg" src="http://192.168.56.22:3001/300/attachments/thumbnail/4" /></a></td>
    </tr>
    <tr>
      <td>image with width</td>
      <td><img src="redmine.jpeg" style="width:60px;" alt="" /></td>
    </tr>
  </table>
```
But thumbnail problem isn't rbpdf's problem.

Please see [Feature #9693 support external url images in PDF](http://www.redmine.org/issues/9693).

**Comment by naitoh** [comment]
fixed 62639d4113153e91e008b566b2b741f315eb4f82
